docs/pages/enroll-resources/desktop-access/troubleshooting.mdx

```diff
@@ -393,3 +393,25 @@ to force the policy update:
 ```code
 $ gpupdate.exe /force
 ```
+
+### CredSSP: server not found in Kerberos database
```

Comment:
Do you think it's also worth adding `CredSSP enhanced RDP security required by server` as a header here, with instructions about setting the environment variable?